planning/behavior_path_planner/src/utils/goal_planner/default_fixed_goal_planner.cpp

```diff
@@ -38,12 +48,63 @@ BehaviorModuleOutput DefaultFixedGoalPlanner::plan(
   return output;
 }
 
+bool isInAnyLane(const lanelet::ConstLanelets & candidate_lanelets, const Point2d & point)
```

Comment:
@HansOersted **[This](https://github.com/autowarefoundation/autoware.universe/blob/dc5d76afeaf4c513458877af46ef7673034ddbd4/planning/behavior_path_planner/src/utils/utils.cpp#L989-L997)** function is useful for you.

Reply:
https://github.com/autowarefoundation/autoware.universe/blob/dc5d76afeaf4c513458877af46ef7673034ddbd4/planning/behavior_path_planner/src/utils/utils.cpp#L989-L997

Reply:
The provided function takes 2 arguments: pose and lanelets.

The function of the current version takes 2 arguments: lanelet and 2Dpoint.
(pose is pre-transferred to 2Dpoint to accommodate the current version,
in reward, it accommodates for publishing the topic of the refined_path <2Dpoint>)

Reply:
replaced by isInLanelets() [https://github.com/autowarefoundation/autoware.universe/commit/a6db1e9f426bdd94629ac48fed0e434a913a167c](https://github.com/autowarefoundation/autoware.universe/commit/a6db1e9f426bdd94629ac48fed0e434a913a167c)

Reply:
@HansOersted 
you can utilize [This](https://github.com/autowarefoundation/autoware.universe/blob/dc5d76afeaf4c513458877af46ef7673034ddbd4/planning/behavior_path_planner/src/utils/utils.cpp#L989-L997)  function by including the file :+1:

Reply:
included the isInLanlets() rather than implement [https://github.com/autowarefoundation/autoware.universe/commit/98d2721056070e93c11ceb46af8429c7fabf3f3d](https://github.com/autowarefoundation/autoware.universe/commit/98d2721056070e93c11ceb46af8429c7fabf3f3d)